src/curve/constant-product/ConstantProductPair.sol

```diff
@@ -167,8 +167,28 @@ contract ConstantProductPair is ReservoirPair {
         amount1 = liquidity * balance1 / _totalSupply; // using balances ensures pro-rata distribution
         require(amount0 > 0 && amount1 > 0, "CP: INSUFFICIENT_LIQ_BURNED");
         _burn(address(this), liquidity);
-        _safeTransfer(_token0, to, amount0);
-        _safeTransfer(_token1, to, amount1);
+
+        if(_safeTransfer(_token0, to, amount0) == false) {
```

Comment:
style

Reply:
you mean make it like this? 
```solidity
if(!_safeTransfer(_token0, to, amount0)) 
```

Actually it was a conscious decision to use `== false` because somehow I think it's really hard to see the exclamation mark in between the `(` and the `_`

Reply:
fixed all